terraform-post-packer/main.tf

```diff
@@ -0,0 +1,79 @@
+# Default AWS provider (EC2AMICreate role in the Images account)
+provider "aws" {
+  region  = "us-east-1"
+  profile = "cool-images-ec2amicreate"
+}
+
+# AWS provider for the Master account (OrganizationsReadOnly role)
+provider "aws" {
+  region  = "us-east-1"
+  profile = "cool-master-organizationsreadonly"
+  alias   = "master"
+}
+
+# Use aws_caller_identity with the default provider (Images account)
+# so we can provide the Images account ID below
+data "aws_caller_identity" "images" {
+}
```

Comment:
Can you please split the providers and caller identity out to `providers.tf` and `caller_identity.tf` so that we are consistent with the code in the `terraform-test-user` directory?

Reply:
Whoops, forgot to do that when I re-added the `terraform-post-packer` sauce. Resolved in be9e283.

Reply:
Looks great! 👍